docs/tutorial/getting_started.md

```diff
@@ -0,0 +1,217 @@
+# Getting started
+
+In this tutorial, we will deploy and run Charmed Chaos Engineering platform using Juju and Terraform.
+
+To complete this tutorial, you will need a machine which meets the following requirements:
+
+- 8GB of RAM
+- 50GB of free disk space
+- An Ubuntu (or another operating system which supports [snapd]) environment to run the commands
+
+## 1. Install Canonical K8s
+
+From your terminal, install Canonical K8s and bootstrap it:
+
+```shell
+sudo snap install k8s --classic --channel=1.33-classic/stable
+cat << EOF | sudo k8s bootstrap --file -
+containerd-base-dir: /opt/containerd
+disable-rbac: true
+cluster-config:
+  network:
+    enabled: true
+  dns:
+    enabled: true
+  local-storage:
+    enabled: true
+EOF
+```
+
+## 2. Bootstrap a Juju controller
+
+From your terminal, install Juju.
```

Comment:
instead of repeating 'from your terminal', we should add a note at the beginning explaining "all the code blocks in this tutorial can be executed from a linux shell"